src/styles.scss

```diff
@@ -1474,3 +1474,31 @@ hr {
 .mat-form-field.small-text {
   font-size: 12px;
 }
+
+// Style the float label of the mat-form-field
+.mat-form-field-appearance-legacy.mat-form-field-can-float.mat-form-field-should-float
```

Comment:
Are these all going to change any other forms elsewhere? Maybe that's the intent if so, but I just want to make sure that other UI isn't being inadvertently changed.

Reply:
Yeah the intention was to change the color of all the float labels to keep it consistent.